docs/api/instruments/minicircuits/usb_2sp2t_dch.rst

```diff
@@ -0,0 +1,7 @@
+####################################################
+Minicircuits 2 Channel DPST USB Solid State RF relay
+####################################################
+
+.. autoclass:: pymeasure.instruments.minicircuits.usb_2sp2t_dch
```

Comment:
`usb_2sp2t_dch` should be capitalized (such that it refers to the actual instrument class, rather than the python file that holds the class)